### PR TITLE
revise the user guide

### PR DESCRIPTION
**Pull request by boyang9527**

### CODE REVIEW

File: docs/Policy_definition.rst [hunk 1]
```diff
@@ -12,30 +12,30 @@ App-AutoScaler Policy Definition
...
-+--------------------------------------+------------------------+---------+----------------------------------------------------+
++--------------------------------------+------------------------+---------+----------------------------------------------------------------+
+| Name                                 | Type                   | Required|Description                                                     |
++--------------------------------------+------------------------+---------+----------------------------------------------------------------+
+| metric_type                          | String                 | true    |one of the following metric types:                              |
+|                                      |                        |         |memoryused,memoryutil,responsetime, throughput                  |
++--------------------------------------+------------------------+---------+----------------------------------------------------------------+
+| stat_window_secs                     | int, seconds           | false   |time interval to avergae metric values                          |
```
Comment (cdlliuy) [inline docs/Policy_definition.rst]: average

File: docs/Readme.md [hunk 2]
```diff
@@ -83,35 +87,33 @@ With [app-autoscaler-cli-plugin][ad] installed, you can manage policy, retrive m
...
 
-* Understand the traffic and workload type of the application
-* Benchmark the application to understand the performance of the application
-* Scale application manually to understand how the application behaves when scaling out ( how long it is needed to warm up, what is the impact of existing load and sticky session if it is used)
+* Benchmark the application to understand the performance
 * Identify the performance bottleneck from the benchmarking result, and decide which metric should be used to dynamically adjust the instance number.
+* Scale application manually to understand how the application behaves when scaling out/in 
 * Define the initial scaling policy and enable the service
-* Simulate the workload and test with the service to adjust detailed settings of the policy, including thresholds, steps of scaling, statistics window, breach duration and cool-down period
+* Drive load and test with the plicy to see how it works. Adjust detailed settings of the policy, including thresholds, steps of scaling, statistics window, breach duration and cool-down period
```
Comment (cdlliuy) [inline docs/Readme.md]: policy

**Comment by boyang9527** [comment]
@cdlliuy  typo fixed.

**Comment by cdlliuy** [comment]
LGTM

**Comment by qibobo** [comment]
LGTM

**Pull request merged**